README.md

```diff
@@ -1,17 +0,0 @@
-# ROS-Cbot
-The C-Bot is an underwater robot for autonomous coral reef monitoring  developed by CSIR National Institue of Oceangraphy, Dona Paula, Goa, India.
-
-#### The C-Bot has following capabilities:
-1. Heading Control
-2. Depth Control
-3. Altitude Control
-4. Way Point Guidance
-5. Path following of straight line and sectors of arc in the presence of oceanic currents
-6. Navigation - Dead Reckoning, Acoustic based use range and bearing from iUSBL and Rnage only based navigation.
-7. Docking uisng acoustics and vision
-8. Capabale of self docking and release can be deployed for 6 months with one survey mission per month, and static monitoring twice a day while docked.
-
-#### Developers:
-1. Pramod Maurya,
-2. Lakshadeep Naik
-3. Mohit Sushil
```

Comment:
Please add a README with detailed instructions for running the software (launch files) including different services used and provided by different components.